os/binfmt/binfmt_execmodule.c

```diff
@@ -128,6 +128,20 @@ static void exec_ctors(FAR void *arg)
 		(*ctor)();
 		ctor++;
 	}
+
+#ifdef CONFIG_SUPPORT_COMMON_BINARY
+	if (g_lib_binp->run_library_ctors) {
+		ctor = g_lib_binp->ctors;
+		for (i = 0; i < g_lib_binp->nctors; i++) {
+			binfo("Calling ctor %d at %p\n", i, (FAR void *)ctor);
+
+			(*ctor)();
+			ctor++;
+		}
+		g_lib_binp->run_library_ctors = false;
+	}
+#endif
```

Comment:
Would there be a particular order in which ctors need to be called?
I think if common binary is present, then classes and variables defined in common binary would be used by the code in app binary. In that case, would it be better to call ctors of common binary first, and the app binary later on?

Reply:
Done.